src/Microsoft.Azure.WebJobs.Shared/Sanitizer.cs

```diff
@@ -2,6 +2,8 @@
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
 using System;
+using System.Linq;
```

Comment:
As part of these changes, I copied over the latest from https://github.com/Azure/azure-functions-host/blob/dev/src/WebJobs.Script/Sanitizer.cs.

Had to make a couple minor changes due to language spec version differences.